src/applications/vaos/tests/new-appointment/components/ContactInfoPage.unit.spec.jsx

```diff
@@ -1,76 +1,83 @@
 import React from 'react';
 import { expect } from 'chai';
 import sinon from 'sinon';
-import { mount } from 'enzyme';
+import userEvent from '@testing-library/user-event';
+import { waitFor } from '@testing-library/dom';
 
-import { fillData } from 'platform/testing/unit/schemaform-utils.jsx';
 import { ContactInfoPage } from '../../../new-appointment/components/ContactInfoPage';
```

Comment:
We want to switch these tests to using the default export from `ContactInfoPage`, so that we test the Redux logic along with the component.

Reply:
Done.